testsuites/syncgateway/upgrade/test_upgrade.py

```diff
@@ -350,17 +393,35 @@ def update_docs(client, ls_url, ls_db, added_docs, auth, terminator_doc_id):
 
     docs_per_update = 3
     doc_revs = {}
+    deleted_docs = []
+
+    user_docs_to_delete = len(current_user_doc_ids) // 10
```

Comment:
you want to remove //10?

Reply:
actually not remove but replace with "threads" variable.